argoverse/data_loading/object_label_record.py

```diff
@@ -204,23 +213,15 @@ def draw_rect(selected_corners: np.array, color: Tuple[int, int, int]) -> None:
         draw_rect(corners[4:], colors[1][::-1])
 
         # grab the top vertices
-        center_top = np.mean(corners[[0,1,4,5]], axis=0)
+        center_top = np.mean(corners[[0, 1, 4, 5]], axis=0)
         uv_ct, _, _, _ = proj_cam_to_uv(center_top.reshape(1, 3), camera_config)
-        uv_ct = uv_ct.squeeze()
+        uv_ct = uv_ct.squeeze().astype(np.int32)  # cast to integer
```

Comment:
This is probably outside the scope of this commit, but I wonder if we should have clear indication of whether we `floor` or `ceil` when converting to discrete coordinates (maybe as a flag to `proj_cam_to_uv`?

Reply:
I think shouldn't matter much, since both floor or ceil can push float uv coords out of the image boundaries. i think closest (round to int), then value check is appropriate. but our `proj_cam_to_uv` is actually really messed up (not vectorized) which we need to fix in another PR